.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:react/recommended',
+    'plugin:jest/style',
+    'pcln-accessibility',
+    'prettier',
+    // 'plugin:@typescript-eslint/eslint-recommended',
+    // 'plugin:@typescript-eslint/recommended',
```

Comment:
ESLint requires a bit of extra configuration to work with TypeScript, so it'll probably make sense to keep these commented out until we support `.tsx?` files outside of the `types` project.

Reply:
sure, makes sense

Reply:
these came as part of `eslint --init` when prompted for TS support. i figured it was easier to add them now and comment it out, then to have to piece it back together later.